src/validation/enums/DomainErrors.enum.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * @enum {DomainErrorMessagesEnum} DomainErrorMessagesEnum - Definition of messages displayed in errors
+ *
+ * @arg {string} InvalidFieldError - Invalid field message error
+ * @arg {string} RequiredFieldError - Required field message error
+ */
+export enum DomainErrorMessagesEnum {
```

Comment:
Rename file and enums to Validation instead of Domain

Reply:
I fix this question @rubemfsv